pepper-apis/src/main/java/org/broadinstitute/ddp/service/KitCheckService.java

```diff
@@ -228,8 +229,8 @@ private boolean handlePrepStep(Handle apisHandle, PendingScheduleRecord pending,
                 pending.getUserId(),
                 pending.getUserId(),
                 pending.getUserGuid(),
-                pending.getStudyId(),
-                EventTriggerType.KIT_PREP);
+                pending.getUserGuid(),
```

Comment:
This service also doesn't have operator.

Reply:
Done